Documentation/Includes.rst.txt

```diff
@@ -0,0 +1,34 @@
+﻿.. More information about this file:
```

Comment:
Imho the file extension is wrong like this, it should be .txt.rst or maybe better .inc.rst to make it correctly recognizable by the IDE as REST file.

Reply:
The file extension `.txt` is used as it is officially announced by the [reStructuredText documentation](https://docutils.sourceforge.io/docs/ref/rst/directives.html#miscellaneous). Some documentations already used the enhanced extension `.rst.txt` which still fits the official documentation but allows the IDE to apply a specific syntax highlighting.

It would probably be no problem to switch finally to `.rst`, but this would be up for a next standardization turn.

Reply:
Tbh we should not follow bad "standards" and the docs do not tell about only using .txt but using it for it's examples. The definition says `The "include" directive reads a text file.` and a `.rst` is also a text file.

> but allows the IDE to apply a specific syntax highlighting

Not sure about a use case here. We put REST inside, nothing special or have I overlooked something?

But this ofc won't be a blocker here.

Reply:
Yes, if you would like to propose .rst to be the new default file extension for include files too, please propose it in an issue at https://github.com/TYPO3-Documentation/T3DocTeam/issues. In this standardization run we are following [this section](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#table-of-contents) which says

> Included reST files have the extension .rst.txt.

As syntax highlighting can be achieved for rst.txt as well as for rst - by applying one additional configuration step in the IDE - i consider this improvement on a low priority, but i understand, that we could discuss it for a next run.